### DISCUSSION TITLE
What to do when the column name is a reserved word of Python

### DISCUSSION THREAD

**Discussion by AzamBukhari** [AammarTufail/python_ka_chilla2023#51] [Q&A]
Hello

During Tips & Tricks lecture we came across a topic of  using a function called: unique()

print(df.embark_town.unique())
![image](https://user-images.githubusercontent.com/121688289/211686299-c665a436-3171-4475-909a-17a25eacc373.png)

What to do if we come across a situation where instead of column name 'embark_town' we have to use 'class' for example:
print(df.class.unique())
![image](https://user-images.githubusercontent.com/121688289/211686510-1c8938b3-a6ea-40a7-9e53-bbe902eef784.png)

it is throwing error.,

**Comment by Masoodullah909** [comment]
df['class'].unique()

**Reply by Muhammad-Ali-Butt** [reply]
i needed this maine tu  class ko qclass rename kr dia titanic k dataset mai aur phir qclass.unique use kia lol
